Modules/Forum/classes/class.ilObjForumGUI.php

```diff
@@ -3312,9 +3302,17 @@ static function (ilForumPost $post) use ($draft): bool {
             $threadContentTemplate->setVariable('TOOLBAR_BOTTOM', $bottom_toolbar->getHTML());
         }
 
-        $this->renderViewModeControl($this->selectedSorting);
-        if ($this->selectedSorting !== ilForumProperties::VIEW_TREE) {
-            $this->renderSortationControl($this->selectedSorting);
+        if ($bottom_toolbar_split_button_items !== []) {
```

Comment:
Maybe we should not use the name `$bottom_toolbar_split_button_items` anymore, if we render the elements to the bottom toolbar *and* the regular toolbar here.